src/analyze/analyze.c

```diff
@@ -2432,6 +2450,28 @@ static int run(int argc, char *argv[]) {
         if (r <= 0)
                 return r;
 
+        /* Functionality to open up and mount the image. Documentation for
+         * what the specific image flags mean can be found on the dissect
+         * header file. */
```

Comment:
This comment is strange. IIUC, "Functionality to …" should be just "…". And "on the dissect header file" means "in dissect-image.h"?